### PR TITLE
Don't prune envoyproxy/go-control-plane

### PR DESCRIPTION
**Pull request by ostromart**

### CODE REVIEW

**Comment by ldemailly** [comment]
hey I had to do some tricks to get "depend.status" to work again,
I apologize do you mind updating/redoing this one ? or maybe I can do it for you if you'd rather not waste time on this

**Review by kyessenov — APPROVED** [review]

File: Gopkg.toml [hunk 1]
```diff
@@ -50,6 +50,10 @@ required = [
     name = "istio.io/api"
     non-go = false
     unused-packages = false
+  [[prune.project]]
+    name = "github.com/envoyproxy/go-control-plane"
```
Comment (ldemailly) [inline Gopkg.toml:54]: 👍

**Comment by ldemailly** [comment]
please update the sha in the other PR

**Comment by ldemailly** [comment]
going in through https://github.com/istio/istio/pull/4059

**Pull request merged**